lib/charms/loki_k8s/v0/loki_push_api.py

```diff
@@ -2132,6 +2140,11 @@ def _scrape_configs(self) -> dict:
                     "label_structured_data": True,
                     "labels": syslog_labels,
                 },
+                "relabel_configs": [
```

Comment:
We should link here to the reference you have [linked to in the pr description](https://github.com/grafana/loki/blob/b24a1430434770a5c4eddc19bfe8e1eddf498a15/pkg/promtail/targets/syslog/syslogtarget.go#L180), for future reference.